### DISCUSSION TITLE
[Feature] Sent 'New media has been added' Notification E-Mail only once

### DISCUSSION THREAD

**Discussion by Green-Kite** [immich-app/immich#12960] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My partner has uploaded over 300 pictures to a shared album and I have received over 300 emails. I would appreciate it if Immich would send a notification only after all pictures have been uploaded and not for each picture individually.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#12424
